app/assets/stylesheets/groups.scss

```diff
@@ -218,13 +218,29 @@
   }
 }
 
+.groups-members-card-avatar {
+  img {
+    height: 60px;
+    padding: 10px 0 8px;
+    transform: translateX(-5px);
+    transition: all .5s;
+    width: 40px;
+
+    &:hover {
+      transform: scale(2.25);
+    }
+  }
+}
+
```

Comment:
```suggestion
.groups-members-card-avatar {
  img {
    border-radius: 50%;
    height: 60px;
    margin-right: 5px;
    padding: 10px 0 8px;
    transform: translateX(-5px);
    transition: all .5s;
    width: 40px;

    &:hover {
      transform: scale(1);
    }
  }
}

```

Reply:
@Arnabdaz for rounded avatar, I had added this with code in 3rd point of the [comment](https://github.com/CircuitVerse/CircuitVerse/pull/3627#issuecomment-1449378729) mentioned above

Reply:
it would not work as I added paddings let me change it to margin and push the updated code

Reply:
![image](https://user-images.githubusercontent.com/96580571/222063258-101a432f-260c-41ef-a52b-41c0d8759ec8.png)
updated member card